src/hooks/useWalletSectionsData.js

```diff
@@ -19,17 +23,15 @@ export default function useWalletSectionsData() {
   const uniswap = useSelector(readableUniswapSelector);
   const { showcaseTokens } = useShowcaseTokens();
 
-  const {
-    currentAction,
-    hiddenCoins,
-    isCoinListEdited,
-    pinnedCoins,
-  } = useCoinListEditOptions();
+  const { hiddenCoins, pinnedCoins } = useCoinListEditOptions();
+  const currentAction = 'none'; // TODO remove this soon
```

Comment:
When can this safely be removed? Should this context be added to the comment?

Reply:
i remvoed this